eng/Version.Details.xml

```diff
@@ -39,113 +39,113 @@
       <Uri>https://github.com/dotnet/aspnetcore</Uri>
       <Sha>ae1a6cbe225b99c0bf38b7e31bf60cb653b73a52</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Primitives" Version="6.0.1-mauipre.1.21602.7" CoherentParentDependency="Microsoft.AspNetCore.App.Runtime.win-x64">
+    <Dependency Name="Microsoft.Extensions.Primitives" Version="6.0.0" CoherentParentDependency="Microsoft.AspNetCore.App.Runtime.win-x64">
```

Comment:
This didn't do the right thing, it downgraded dotnet/runtime?

Reply:
After fixing `<Dependency Name="Microsoft.NETCore.App.Ref" Version="6.0.1-mauipre.1.21616.2" CoherentParentDependency="Microsoft.Android.Sdk.Windows">`, I think this is actually correct.